addons/cloud-games/addon.json

```diff
@@ -11,8 +11,25 @@
   ],
   "popup": {
     "icon": "../../images/icons/cloud.svg",
-    "name": "Games"
+    "name": "Games",
+    "script": "popup.js",
+    "html": "popup.html"
   },
+  "settings": [
+    {
+      "name": "Studio URL",
+      "id": "url",
+      "type": "string",
+      "default": "https://scratch.mit.edu/studios/539952/"
```

Comment:
Just id maybe?

Reply:
Maybe. `url` is fine IMO, though.

Reply:
Only scratch urls work, so why not just shorten it to the id?

Reply:
it's user-facing, and the user is more likely to know what the url is

but if we describe the id to them, they could probably be able to find it. plus that'd make it so we don't have to worry about them ommitting the trailing slash, or adding /projects to the end or using non scratch URLs

i actually like that idea better

Reply:
Note that while this input says "Studio URL", any string that has digits inside (including plain studio ID) works.

Reply:
Maybe `STUDIO URL/ID`

Reply:
Studio URL works IMO. In most cases they'll get an error if you put random words.